src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,116 @@
+
+.card {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  max-width: 200px;
+  max-height: 408px;
+  padding: 32px 16px 16px;
+
+  @extend %font-page-roboto500;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__img-box {
+    display: flex;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    width: 100%;
+
+    font-size: 12px;
+    @extend %font-page-roboto500;
+    line-height: 18px;
+    color: $color-text;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    align-self: flex-start;
+
+    font-size: 10px;
+    @extend %font-page-roboto400;
+    line-height: 14px;
+    color: $color-text-card__small;
+  }
+
+  &__review.review {
+    margin-bottom: 24px;
+
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    .review__text {
+      font-style: normal;
+      font-size: 10px;
+      line-height: 14px;
+      @extend %font-page-roboto400;
+      color: $color-text;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    width: 166px;
+    box-sizing: border-box;
+
+    text-decoration: none;
+    color: #fff;
+    font-size: 14px;
+    line-height: 16px;
+    @extend %font-page-roboto700;
+    text-transform: uppercase;
+
+    background: $color-blue-link;
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    &:hover {
+      background: #fff;
+      border-color: $color-blue-link;
+      color: $color-blue-link;
+    }
+  }
+}
+// ! leave this element outside to get mentor review in detail, please!
+// Q: what is more important: decreasing specificity or BEM structure?
```

Comment:
each bem block should have its own file
does this answers your question?
and better to ask questions like this in qna channel or your private channel, instead of a code :)

Reply:
ok)